src/DependencyInjection/Configuration.php

```diff
@@ -43,29 +44,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                             ->defaultValue('App')
                             ->cannotBeEmpty()
                         ->end()
-                        ->arrayNode('patterns')
-                            ->children()
-                                ->arrayNode('directories')
-                                    ->info('Directories to parse (supports pattern syntax)')
-                                    ->scalarPrototype()->end()
-                                    ->defaultValue([''])
-                                ->end()
-                                ->arrayNode('names')
-                                    ->info('File names to parse (supports pattern syntax)')
-                                    ->scalarPrototype()->end()
-                                    ->defaultValue(['*.php'])
-                                ->end()
-                                ->arrayNode('exclude')
-                                    ->info('Files or directories to ignore (supports pattern syntax)')
-                                    ->scalarPrototype()->end()
-                                    ->defaultValue([])
-                                ->end()
-                                ->arrayNode('class_tags_to_ignore')
-                                    ->info('PHP tags to ignore')
-                                    ->scalarPrototype()->end()
-                                    ->defaultValue(['@internal', '@experimental'])
-                                ->end()
-                            ->end()
+                        ->arrayNode('exclude')
+                            ->info('Files or directories to ignore (supports pattern syntax)')
```

Comment:
`exclude` is used in `Finder::notName` method call which doesn't support directories, only files name matching a pattern